src/eko/evolution_operator/grid.py

```diff
@@ -11,11 +11,10 @@
 
 import numpy as np
 
-from eko import matching_conditions, member
-from eko.evolution_operator import flavors
```

Comment:
Good, this is an old style import, leftover since ages. There are still a couple around, could you fix them as well? Thanks :)
```py

beta.py
10:from eko import constants

anomalous_dimensions/lo.py
7:from eko import constants
```

Reply:
done in [8827cc7](https://github.com/N3PDF/eko/pull/89/commits/8827cc7123e8c721c8b7142db574a1e1983428db)